ext/LegendDataManagementSolidStateDetectorsExt.jl

```diff
@@ -440,7 +443,9 @@ function create_SSD_config_dict_from_LEGEND_metadata(meta::PropDict, xtal_meta::
 
 
     ### MANTLE CONTACT ###
-    Vop, Vop_val_used = if hasproperty(meta.characterization.l200_site, :recommended_voltage_in_V) && meta.characterization.l200_site.recommended_voltage_in_V > 0
+    Vop, Vop_val_used = if ustrip(operational_voltage) >= 0
+        operational_voltage isa Unitful.Quantity ? ustrip(u"V", operational_voltage) : operational_voltage, "✔ Operational voltage (user):"
+    elseif hasproperty(meta.characterization.l200_site, :recommended_voltage_in_V) && meta.characterization.l200_site.recommended_voltage_in_V > 0
```

Comment:
Maybe explicitly require `Vop, Vop_val_used` to be `Float` to avoid them being `Rational`.
```suggestion
    Vop::Float64, Vop_val_used::Float64 = if ustrip(operational_voltage) >= 0
        operational_voltage isa Unitful.Quantity ? ustrip(u"V", operational_voltage) : operational_voltage, "✔ Operational voltage (user):"
    elseif hasproperty(meta.characterization.l200_site, :recommended_voltage_in_V) && meta.characterization.l200_site.recommended_voltage_in_V > 0
```

or apply `float` here already?
```suggestion
    Vop, Vop_val_used = if ustrip(operational_voltage) >= 0
        operational_voltage isa Unitful.Quantity ? float(ustrip(u"V", operational_voltage)) : operational_voltage, "✔ Operational voltage (user):"
    elseif hasproperty(meta.characterization.l200_site, :recommended_voltage_in_V) && meta.characterization.l200_site.recommended_voltage_in_V > 0
```

Reply:
No need, it only needs to be a non-rational when printing output